components/dropbox/sources/new-folder/new-folder.mjs

```diff
@@ -0,0 +1,17 @@
+import common from "../common.mjs";
+
+export default {
+  ...common,
+  key: "dropbox-new-folder",
+  name: "New Folder",
+  version: "0.0.5",
+  description: "Emits an event when a new folder is created. Make sure the number of files/folders in the watched folder does not exceed 4000.",
+  async run() {
```

Comment:
The eslint rules like it if source descriptions start with "Emit new".
`description: "Emit new event when a new folder is created. Make sure the number of files/folders in the watched folder does not exceed 4000.",`

Also, there should be a type here:
`type: "source",`